src/Accordion.js

```diff
@@ -42,17 +41,15 @@ const Accordion = React.forwardRef((props, ref) => {
   bsPrefix = useBootstrapPrefix(bsPrefix, 'accordion');
 
   return (
-    <AccordionContext.Provider value={{ onClick, activeEventKey: activeKey }}>
-      <SelectableContext.Provider value={null}>
-        <Component
-          ref={ref}
-          {...controlledProps}
-          className={classNames(className, bsPrefix)}
-        >
-          {children}
-        </Component>
-      </SelectableContext.Provider>
-    </AccordionContext.Provider>
+    <SelectableContext.Provider value={{ onClick, activeEventKey: activeKey }}>
```

Comment:
if you look at the other use cases, we only use `<SelectableContext.Provider>` to pass down `onSelect`, while using a different context to pass down details of the active item. i guess because the latter varies from place to place?

so to follow that, we would want both contexts, but we'd want to use them differently

Reply:
Ah. So I should use AccordionContext to pass down the current active key and then use SelectableContext to pass down onSelect?

Reply:
yeah – that's what we do elsewhere. it's a little weird but better to be consistent, i think.